platform/stm32f0xx/timer_capture.c

```diff
@@ -329,7 +331,9 @@ status_t stm32_timer_capture_setup(stm32_timer_capture_t *tc, int timer, uint16_
     spin_lock_init(&tc->overflow_lock);
 
     tc->config->regs->CR1 = 0;
-    tc->config->regs->PSC = prescaler;
+
+    // The prescaler value applied by hardware is PSC + 1.
+    tc->config->regs->PSC = prescaler > 0 ? prescaler - 1 : 0;
```

Comment:
Curious why we have both the assert for prescaler > 0 at the top of the function as well as the check here.

Reply:
It's a habit I got into at Makani -- assert on your expectations, but then try to do something reasonable if they fail in case the code is compiled without asserts.

Reply:
Yah I generally do the same thing. LGTM.

Reply:
Thanks, Travis!